### PR TITLE
fixed some deprecated notices

### PR DESCRIPTION
**Pull request by fabpot**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | no |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

**Comment by stof** [comment]
:+1:

File: src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php [hunk 1]
```diff
@@ -63,6 +64,10 @@ public function process(ContainerBuilder $container)
             $container->getDefinition('twig.extension.code')->replaceArgument(0, $container->getParameter('templating.helper.code.file_link_format'));
         }
 
+        if (!$container->has('security.token_storage')) {
```
Comment (xabbuh) [inline src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php]: Are you sure? Shouldn't this be `if ($container->has('security.token_storage'))` (without the `!`)?

Comment (fabpot) [inline src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php]: I'm sure :) If the token storage does not exist, it means you are using an older version, where we need to inject the security service.

Comment (xabbuh) [inline src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php]: Ah, now I get what you are trying to achieve. But then, the service definition requested via `getDefinition()` below should be `twig.app_variable` and not `security.token_storage`.

Comment (fabpot) [inline src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php]: oops

**Pull request merged**